CONTRIBUTING

```diff
@@ -1,26 +1,26 @@
-HOW TO CONTRIBUTE PATCHES TO OpenSSL
+HOW TO CONTRIBUTE TO PATCHES OpenSSL
```

Comment:
Did you change this line intentionally? This does not sound like correct english to me.

Reply:
Maybe you intended to write "HOW TO CONTRIBUTE TO OpenSSL"?